cli/generators/crud/index.js

```diff
@@ -7,6 +7,13 @@ module.exports = class extends Generator {
     super(args, opts);
 
     this.argument('name', { type: String, required: true });
+
+    this.option('api', {
```

Comment:
No. The `no-` prefix is like the negative version of your option. I didn't invest time on researching it but I believe is on purpose so you can use `--api` or `--no-api`. Actually I first tried with `no-api` but it doesn't work. I believe that if I dig a little bit on the documentation I will find the explanation...